README.md

```diff
@@ -498,3 +500,9 @@ If you want to clean artifacts of all contexts, you should to run `clean --curre
 * `eden utils gcp` - sub-commands to work with Google Cloud Platform
 * `eden utils export` - sub-command to save certs and configs into tar.gz
 * `eden utils import` - sub-commands to load certs and configs into tar.gz
+
+## Parallels
+
+0. Shut down the virtual machine before changing the hypervisor type.
+1. Open virtual machine configuration > **Hardware > CPU & Memory > Advanced Settings**, and set Hypervisor type to **Parallels**.
+2. Click the **Enable Nested Virtualization** checkbox.
```

Comment:
The description is not enough. Please provide a detailed one. 
You need to consider somebody who wants to use eden +eve on Mac with ability to launch VM's in EVE. 
Your instruction is not clear.  Also you may want to add pictures

Should start like: 
If you want to be able to launch VMs or containers-in-VMs or accel=true on Mac 
You need to use Parallels. 
1) Set up parallels ... 
2) 
3)